data/friday_hacks.yml

```diff
@@ -36,8 +28,8 @@ hacks:
         title: TBA
       - speaker: Sheng You
         title: Ktor in Action
-  - nohack: true
-  - nohack: true
+  - nohack: Recess Week
+  - nohack: Exam Week
   - venue: TBD (Physical)
```

Comment:
really minor nitpick but physical probably doesn't need to be indicated, since it is TBD anyway

Reply:
Yeah sure!